packages/x-charts/src/models/axis.ts

```diff
@@ -336,11 +336,35 @@ export type AxisValueFormatterContext<S extends ScaleName = ScaleName> =
        * - `'legend'` The value is displayed in the legend when using color legend.
        * - `'zoom-slider-tooltip'` The value is displayed in the zoom slider tooltip.
        */
-      location: 'tick' | 'tooltip' | 'zoom-slider-tooltip';
+      location: 'tooltip' | 'zoom-slider-tooltip';
       /**
        * The d3-scale instance associated to the axis.
        */
       scale: AxisScaleConfig[S]['scale'];
+    }
+  | {
+      /**
+       * Location indicates where the value will be displayed.
+       * - `'tick'` The value is displayed on the axis ticks.
+       * - `'tooltip'` The value is displayed in the tooltip when hovering the chart.
+       * - `'legend'` The value is displayed in the legend when using color legend.
+       * - `'zoom-slider-tooltip'` The value is displayed in the zoom slider tooltip.
+       */
+      location: 'tick';
+      /**
+       * The d3-scale instance associated to the axis.
+       */
+      scale: AxisScaleConfig[S]['scale'];
+      /**
+       * The tick label shown by default if the value isn't formatted.
```

Comment:
```suggestion
       * The tick label shown by default if the value isn't formatted.
       * With log scale it can be an empty string to display ticks without labels.
       * @see See {@link https://d3js.org/d3-scale/log#log_tickFormat D3 log scale docs} for more details.```